.gitignore

```diff
@@ -360,4 +360,5 @@ build/
 dist/
```

Comment:
@HazemSaeid Please remove this file from this PR

Reply:
Hi, sorry for the late answer. The .gitignore file is now deleted.